swagger.json

```diff
@@ -15,49 +15,52 @@
   "paths" : {
     "/resource" : {
       "post" : {
-        "summary" : "Deposit a new resource into SDR.",
-        "description" : "Deposits a new resource (Collection, Digital Repository Object, Fileset, or subclass of those) into SDR. Will return the SDR identifier for the resource.",
+        "summary" : "Deposit New TACO Resource.",
+        "description" : "Deposits a new resource (Collection, Digital Repository Object, File [metadata only] or subclass of those) into SDR. Will return the SDR identifier for the resource.",
         "operationId" : "depositNewResource",
-        "consumes" : [ "application/json" ],
+        "consumes" : [ "application/json", "application/json+ld" ],
         "produces" : [ "application/json" ],
         "parameters" : [ {
           "in" : "body",
           "name" : "payload",
-          "description" : "JSON-LD Representation of the resource metadata going into SDR. Needs to fit the SDR 3.0 MAP requirements.",
+          "description" : "JSON-LD representation of the resource metadata going into SDR. Needs to fit the SDR 3.0 MAP requirements.",
           "required" : true,
           "schema" : {
             "$ref" : "#/definitions/Resource"
           }
         } ],
         "responses" : {
-          "200" : {
-            "description" : "Success response",
+          "201" : {
+            "description" : "TACO resource created & processing started.",
             "schema" : {
-              "type" : "object",
-              "properties" : {
-                "id" : {
-                  "type" : "string",
-                  "example" : "oo000oo0001"
-                }
-              }
+              "$ref": "#/definitions/ResourceResponse"
             }
           },
-          "405" : {
-            "description" : "Invalid input"
+          "401" : {
+            "description" : "You are not authorized to create a resource in TACO."
+          },
+          "415" : {
+            "description" : "Unsupported resource type provided. TACO resources should be handed over as JSON or JSON-LD."
+          },
+          "422" : {
+            "description" : "The resource JSON provided had an unspecified or unsupported field, or is otherwise unprocessable by TACO."
+          },
+          "500" : {
```

Comment:
Can the 4xx/5xx responses have a schema so we can give the user helpful error messages?

Reply:
we haven't gotten there yet. we need to talk exception handling per today's decisions.